autometa/common/external/hmmer.py

```diff
@@ -138,18 +145,25 @@ def hmmscan(orfs, hmmdb, outfpath, cpus=0, force=False, parallel=True):
 
     Raises
     -------
+    ValueError
+        Both `parallel` and `gnu_parallel` were provided as True
     FileExistsError
         `outfpath` already exists
     subprocess.CalledProcessError
         hmmscan failed
```

Comment:
I don't see this error being raised here
```suggestion
```

Reply:
The functions called within this are raising the `subproces.CalledProcessError`